plugins/parsers/json/README.md

```diff
@@ -20,7 +20,7 @@ ignored unless specified in the `tag_key` or `json_string_fields` options.
 
   ## When strict is true and a JSON array is being parsed, all objects within the
   ## array must be valid
-  strict = false
+  strict = true
```

Comment:
Doesn't this need to be named json_strict?

Reply:
Yes, thanks.